### ISSUE TITLE
Add reason for excused absence into late days table

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#10123]
**What problem are you trying to solve with Submitty**
We recently added an optional string "reason" for each "excused absence extension"

We already display that string in rainbow grades.  But let's also add it to the late days table:
<img width="278" alt="Screenshot 2024-01-22 at 9 47 57 PM" src="https://github.com/Submitty/Submitty/assets/5871417/4378abc7-03cb-486a-86eb-d9729a3b55ed">

So in the "granted # of days extension" column, instead of just "1", let's make it for example "1 (illness)"
This is displayed in the ta grading interface in the "student information" panel.
And also when the student clicks on "my late days/extensions" from the left sidebar

**Comment by niteshbca** [comment]
I am a solve is issue

**Comment by bmcutler** [comment]
Hi @niteshbca 

Thank you for your interest in the Submitty open source project.
We welcome contributions from new developers!
However we do not use the Github issue 'assign' feature for first time prospective contributors.

Please read our documentation on how to to get started with Submitty:
https://submitty.org/developer/getting_started/index
Specifically in setting up your development environment:
https://submitty.org/developer/getting_started/vm_install_using_vagrant
And making a pull request:
https://submitty.org/developer/getting_started/make_a_pull_request
You do not need to be assigned to an issue to create a pull request that will be reviewed by our team and then merged if it appropriately resolves the issue.
We also encourage you to join our Zulip server to discuss technical questions:
https://submitty.org/index/contact

**Comment by niteshbca** [comment]
Sir is problem ka salutation ye hai
Python me
# Update the late days table display function to include the reason string
def display_late_days_table(student_id): late_days =
get_late_days(student_id) for entry in late_days: reason = entry.get(
'reason', '') days = entry['days'] print(f"{days} ({reason})")
Please sir right hai to yes ya no answer dijiye please 👃👃

**Comment by CCWizma99** [comment]
Is this issue was closed by @Xiayucheng1212? When I compare my code in LateDaysTablePlugin.twig with the mentioned contributor, It seems like both codes are identical to naked eye. Anyways, I made a pull request with the changes to show the reason for excused absence. 

Here's my PR:
[[UI/UX:InstructorUI] Late days exception UI improvement](https://github.com/Submitty/Submitty/pull/10294)

Thank you...
